index.html

```diff
@@ -446,9 +446,9 @@ <h3>
         </h3>
         <p>
           The [=manifest's=] <code><dfn data-export="" data-dfn-for=
-          "manifest">dir</dfn></code> member specifies the <dfn>base
-          direction</dfn> for the <a>localizable members</a> of the
-          <a>manifest</a>. The [=manifest/dir=] member's value can be set to a
+          "manifest">dir</dfn></code> member specifies the string direction for
```

Comment:
I would probably say that this is the _default_ string direction, since individual members can override the direction. I would probably also linkify the term "string direction" (to the I18N glossary) to ensure that the meaning is clear.

```suggestion
          "manifest">dir</dfn></code> member specifies the default [=string direction=] for
```

Reply:
@christianliebel 

> @aphillips I’m unsure if (and from where) string direction and first-strong detection should be cited.

All of I18N's definitions are in the I18N glossary, which is set up for easy linkification in any Respec spec. We try to provide [instructions](https://www.w3.org/TR/i18n-glossary/#respec-how-to-use)